syntax = "proto3";

package calc;

service CalcService{
  rpc Sum(SumElements) returns (SumResponse);
  rpc DelayHello(time) returns (HelloResponse);
}

message SumElements {
  repeated int64 elem = 1;
}

message SumResponse{
  int64 result = 1;
}

message time{
  int64 time = 1;
}

message HelloResponse{
  string msg = 1;
}
